gatsby/on-route-update.js

```diff
@@ -1,10 +1,10 @@
 const onRouteUpdate = ({ location, prevLocation }) => {
   if (
-    window.Tessen &&
+    window.newrelic &&
     location.hash &&
     location.pathname !== prevLocation?.pathname
   ) {
-    window.Tessen.track({
+    window.newrelic.track({
```

Comment:
Does this need to be Segment or Browser Agent? If its browser agent, it could either import the `addPageAction` function or use the global but with `.addPageAction` vs `.track`

Reply:
we won't be using segment in these sites outside of setting the config options. It should be browser agent

Reply:
lets use the browser agent method `addCustomAttribute()`!

Reply:
This has been added and we can see `hashRequest` values in browser events now.